tst/regression/utils/test_case.py

```diff
@@ -182,6 +183,9 @@ def MakeOutputFolder(self):
         if not os.path.isdir(self.parameters.output_path):
             os.makedirs(self.parameters.output_path)
 
+        files = glob.glob(self.parameters.output_path + "/*")
+        for f in files:
+            os.remove(f)
```

Comment:
Could we potentially only clean the required files for specific tests, e.g., the `advection_errors.dat` for the advection test?
I always get goosebumps if automated pattern matching with a `*` is used to remove files in scripts/programs.

Reply:
Where would you put the logic?

Reply:
How about directly in the Prepare function of each test case like
```Python
if step == 1:
  magic_cleanup()
```

Reply:
@pgrete I can see having a clean output folder being a good default. How about I simply remove the named output folder, that will avoid the wildcard. We could always have a setting later that turns this default feature off.

Reply:
@pgrete

Reply:
Sounds good. My concern was mostly about the wildcard itself ;)